ospd_openvas/daemon.py

```diff
@@ -717,9 +717,9 @@ def get_dependencies_vt_as_xml_str(
         vt_deps_xml = Element('dependencies')
         for dep in vt_dependencies:
             _vt_dep = Element('dependency')
-            try:
+            if "1.3.6.1.4.1.25623." in dep:
```

Comment:
Could you please add a global constant for this OID value. From the outside this value is very difficult to understand.